### PR TITLE
Update howto for finetuning with adapter

### PR DESCRIPTION
**Pull request by awaelchli**
Requires #31

### CODE REVIEW

**Review by carmocca — APPROVED** [review]
Thanks!

BTW I removed the quantize sections because it's not working yet: #18

File: howto/finetune_adapter.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ or [prepare your own dataset](#tune-on-your-dataset).
 python finetune_adapter.py --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b
 ```
 
+The finetuning requires at least one GPU with ~24 GB memory (GTX 3090).
```
Comment (carmocca) [inline howto/finetune_adapter.md:30]: Is this number updated for the 3b stablelm model?

Comment (ArturK-85) [reply howto/finetune_adapter.md]: with micro_batch_size = 4

Comment (carmocca) [reply howto/finetune_adapter.md]: Thank you!

**Review by lantiga — APPROVED** [review]
Looks great

**Pull request merged**